cockatrice/src/dlg_add_set.cpp

```diff
@@ -0,0 +1,37 @@
+#include "dlg_add_set.h"
+
+#include <QDialogButtonBox>
+#include <QPushButton>
+#include <QVBoxLayout>
+
+DlgAddSet::DlgAddSet(QWidget *parent, bool success) : QDialog(parent) {
+    status = new QLabel(this);
+    restart = new QLabel(this);
+
+    if (success) {
+        setWindowTitle(tr("Success"));
+        status->setText(QString("Set added to Cockatrice."));
+        restart->setText(QString("You must restart to use your new set."));
```

Comment:
I'll try to review this tonight. Overall a needed change

On Sat, Feb 6, 2016, 3:25 AM tooomm <notifications@github.com> wrote:

> In cockatrice/src/dlg_add_set.cpp
> <https://github.com/Cockatrice/Cockatrice/pull/1796#discussion_r52099762>:
>
> > @@ -0,0 +1,37 @@
> > +#include "dlg_add_set.h"
> > +
> > +#include <QDialogButtonBox>
> > +#include <QPushButton>
> > +#include <QVBoxLayout>
> > +
> > +DlgAddSet::DlgAddSet(QWidget *parent, bool success) : QDialog(parent) {
> > +    status = new QLabel(this);
> > +    restart = new QLabel(this);
> > +
> > +    if (success) {
> > +        setWindowTitle(tr("Success"));
> > +        status->setText(QString("Set added to Cockatrice."));
> > +        restart->setText(QString("You must restart to use your new set."));
>
> "You must restart *Cockatrice* to use *your* new set."
> Otherwise some people might want to restart their PC... :)
>
> I would prefer "the" over "your", too.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Cockatrice/Cockatrice/pull/1796/files#r52099762>.
>